docker-compose.dev.yml

```
@@ -185,8 +190,83 @@ services:
       graph-db:
         condition: service_healthy
 
+  bh-authentik:
+    profiles:
+      - sso
+      - sso-only
+    image: ${AUTHENTIK_IMAGE:-ghcr.io/goauthentik/server}:${AUTHENTIK_TAG:-2023.10.7}
+    restart: unless-stopped
+    command: server
+    environment:
+      AUTHENTIK_REDIS__HOST: bh-authentik-valkey
+      AUTHENTIK_POSTGRESQL__HOST: bh-authentik-db
+      AUTHENTIK_POSTGRESQL__USER: ${BH_PG_USER:-authentik-bloodhound}
+      AUTHENTIK_POSTGRESQL__NAME: ${BH_PG_DB:-authentik-bloodhound}
```

I think we can keep these just authentik
```suggestion
      AUTHENTIK_POSTGRESQL__USER: ${BH_PG_USER:-authentik}
      AUTHENTIK_POSTGRESQL__NAME: ${BH_PG_DB:-authentik}
```